### ISSUE TITLE
Use a more specific name instead `attributes` for `import`.

### ISSUE THREAD

**Issue by SidOfc** [SidOfc/leather#14]
Oh and last feedback for now: I find `attributes` a bit strange and too generic as an import that one would use in the context of a broader project, because it does not indicate that it's a method reading something or returning something. It sounds like the name of a local variable. I therefore import  it like this:

```js
import { attributes as readMediaAttributes } from 'leather'
```

Something to think about...

_Originally posted by @lehni in https://github.com/SidOfc/leather/issues/9#issuecomment-1477537520_

**Comment by SidOfc** [comment]
I will use `readMediaAttributes` from the original comment as-is since I think it's much better than the generic `attributes` name. Additionally I'm thinking of making this a breaking change and bumping the major version instead of doing it in a minor version with deprecation notice.

**Comment by lehni** [comment]
…or just default export? And then people can import as whatever they want, e.g. `import leather from 'leather;`

**Comment by SidOfc** [comment]
@lehni I'm using rollup to build both a cjs and esm build, rollup states that there are some issues when trying to use default exports as explained here: https://rollupjs.org/configuration-options/#output-exports

Basically all is fine as long as I'd _only_ have a default export but mixing default with named exports in both cjs and esm would become a bit messy. I do not want to limit myself to this as the future may hold more functionality and exports so I prefer using named only.

A default cjs export is always available by using:

```js
const leather = require('leather');
```

And in esm this can be emulated like so:

```js
import * as leather from 'leather';
```

Even if default export would be used instead, I still think it makes sense to rename the generic `attributes` to the name you originally proposed anyway for clarity :)

**Comment by lehni** [comment]
Yes makes perfect sense!

**Comment by SidOfc** [comment]
@lehni just released [version 3.0.0](https://github.com/SidOfc/leather/releases/tag/3.0.0) which renames `attributes` to `readMediaAttributes`. Going to close this since it's resolved. Cheers for the feedback 👍

**Comment by lehni** [comment]
@SidOfc works like a charm, thanks!
